public/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Testing the new Bike Index Widget</title>
+  </head>
+  <body>
```

Comment:
Could you add some styles here so that the index page look as nice as the current https://widget.bikeindex.org/

<img width="835" alt="Screen Shot 2019-03-25 at 11 33 48 AM" src="https://user-images.githubusercontent.com/1235441/54944946-e9511200-4ef1-11e9-9554-1995336420e2.png">

favicon, centered, etc.

Reply:
Sounds good.  I just added something to preserve the previous styles, etc